includes/discovery/ports/port-xdsl.inc.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+use LibreNMS\Config;
+
+if (Config::get('enable_ports_adsl')) {
```

Comment:
enable_ports_adsl feels like an odd barrier to usage.  Maybe os should define to enable adsl discovery or not.
Or maybe it should even be its own module?

Reply:
I think I'll get rid of enable_ports_adsl completely. And adapt the documentation indeed. There is no reason to keep it now that discovery does the "messy" part (like polling slow and not existing OIDs) and poller only retrieve the data. 

And as discussed on Discord, will see if new dedicated module makes sense.

Reply:
You convinced me I guess :)